hy/compiler.py

```diff
@@ -559,9 +580,11 @@ def compile_unpack_iterable(self, expr, root, arg):
         return ret
 
     @special([(not PY3, "exec*")], [FORM, maybe(FORM), maybe(FORM)])
-    # Under Python 3, `exec` is a function rather than a statement type, so Hy
-    # doesn't need a special form for it.
     def compile_exec(self, expr, root, body, globals_, locals_):
+        """
+        Under Python 3, `exec` is a function rather than a statement type,
+        so Hy doesn't need a special form for it.
+        """
         return asty.Exec(
```

Comment:
Why's this conversation marked resolved? Do you feel that the deleted comment is unhelpful? I added it originally because I figured it wasn't obvious why the method was gated to `not PY3`. But whether we end up removing the comment or not, removing it seems beside the point of this commit.

Reply:
Sorry, I think I've just been confused about your request: you want it there in its original form, and not removed?

Reply:
Right. Sorry, I was sloppy.

Reply:
Ohh, looks like they just added this "insert a suggestion" feature (literally just popped up as I wrote the previous comment).  If it shows up for you, let's try it!

Reply:
"Applying suggestions on deleted lines is not supported." :(